pkg/rebuild/rebuild/storage.go

```diff
@@ -87,18 +89,24 @@ type LocatableAssetStore interface {
 	URL(a Asset) *url.URL
 }
 
+func logWrap(f func() error) {
+	if err := f(); err != nil {
+		log.Print(err)
+	}
+}
+
 // AssetCopy copies an asset from one store to another.
 func AssetCopy(ctx context.Context, to, from AssetStore, a Asset) error {
 	r, err := from.Reader(ctx, a)
 	if err != nil {
 		return errors.Wrap(err, "from.Reader failed")
 	}
-	defer r.Close()
+	defer logWrap(r.Close)
```

Comment:
do we need this? I'd like to avoid fire-and-forget logging like this. would it be possible to tie this in to the return value?